esx_service/cli/vmdkops_admin.py

```diff
@@ -955,9 +958,13 @@ def tenant_access_set(args):
     if args.volume_totalsize:
         volume_totalsize_in_MB = convert.convert_to_MB(args.volume_totalsize)
 
+    if not validate_create_value(str(args.allow_create)):
```

Comment:
I agree with your first point.
Regarding second point, for the tenant_access_add method, the command line behavior is such that mentioning --allow-create option itself means the option is true. Not mentioning it means it is false. So there isn't a scope for invalid input to be received by tenant_access_add method through Admin CLI interface.
But I am not sure when this api is called through VMODL API; should we assume the caller of tenant_access_add() through VMODL gives a valid value for allow-create option; or is there a scope to receive a wrong value for it?